app/controllers/users_controller.rb

```diff
@@ -9,6 +9,7 @@ def destroy
 
   def destroy_user
     user = current_user || User.find_by(confirmation_token: session[:user_confirmation_token])
+    session.delete(:user_confirmation_token)
```

Comment:
is there some reason we wouldn't just destroy the entire session (log the user out) at this point?